server/text_generation_server/models/globals.py

```diff
@@ -25,3 +25,14 @@
 def set_model_id(model_id: str):
     global MODEL_ID
     MODEL_ID = model_id
+
+
+# NOTE: eventually we should move this into the router and pass back the
+# index in all cases.
+global ADAPTER_TO_INDEX
+ADAPTER_TO_INDEX = None
```

Comment:
```suggestion
ADAPTER_TO_INDEX: Dict[str, int] = None
```

(Not sure if it's the correct type.)

Reply:
thats correct and updated!